src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -71,7 +74,49 @@ function FlipperClient({
       }
     }
     */
-    const { data } = await _fetchToggleValues();
+    let data;
+    const queryParams = new URLSearchParams(window.location.search);
+    const isToggleCacheDisabled =
+      queryParams.get('disableFlipperCache') === 'true';
+    const isPostLogin = queryParams.get('postLogin') === 'true';
+    const signOutFlag = sessionStorage.getItem('signOut') === 'true';
+
+    const featureToggleSessionData =
+      sessionStorage.getItem(TOGGLE_STORAGE_KEY) &&
+      JSON.parse(sessionStorage.getItem(TOGGLE_STORAGE_KEY));
+
+    const isSessionDataValid =
+      featureToggleSessionData &&
+      Date.now() < new Date(featureToggleSessionData.expiresAt).getTime();
+
+    if (signOutFlag) {
+      setTimeout(() => {
+        sessionStorage.removeItem('signOut');
```

Comment:
Because we just want to make a feature toggle call when sign out is triggered, as it is of no use after that I thought of removing it.

Other option is to set it to `false`